AddonArmTemplate/FullAzureMonitorMetricsProfile.json

```diff
@@ -462,7 +465,8 @@
       "properties": {
         "description": "[concat(variables('RecordingRuleGroupDescriptionWin'), variables('version'))]",
         "scopes": [
-          "[parameters('azureMonitorWorkspaceResourceId')]"
```

Comment:
Will you also be making a change to the Alerts RP from where the CLI is downloading the recording rule groups and also add this to the CLI recording rule scope -> [here](https://github.com/Azure/azure-cli/blob/dev/src/azure-cli/azure/cli/command_modules/acs/azuremonitormetrics/recordingrules/create.py#L31)?

Also we would need to make a similar change in the UX (Greenfield + Brownfield), can you sync with them too?

Reply:
@Sohamdg081992 I am making changes in the CLI for a bug fix. I can add this scope update in that PR.

Reply:
Sounds good. Thanks Kaveesh!